src/webfaf/templates/problems/list.html

```diff
@@ -43,7 +43,15 @@
                 {{ render_field(filter_form.since_release, placeholder="Since release") }}
                 {{ render_field(filter_form.to_version, placeholder="To version") }}
                 {{ render_field(filter_form.to_release, placeholder="To release") }}
-                <p class="text-muted">Version range</p>
+                <p class="text-muted">Version range
+                    <a role="button" data-toggle="tooltip" title="To search in bugs for component foo within versions &quot;1.1.1-1&quot; and &quot;2.0.0-3&quot;, set
```

Comment:
This worked for me:
```data-toggle="tooltip" data-html="true" ...```

and 
``` js
<script type="text/javascript">
    $(document).ready(function(){
        $('a[data-toggle=tooltip]').tooltip();
    });
</script>
```

If you tried this before and it didn't work, it might have been caused by 40ada9c17949fb71a2c417f7d9a67de1fbf512af.

Reply:
Thank you, it works now.